dev/bots/run_command.dart

```diff
@@ -190,13 +190,24 @@ Future<CommandResult> runCommand(String executable, List<String> arguments, {
         print(result.flattenedStderr);
         break;
     }
+    String allOutput;
+    if (failureMessage == null) {
+      allOutput = '${result.flattenedStdout}\n${result.flattenedStderr}';
+      if (allOutput.split('\n').length > 10) {
```

Comment:
What about (couldn't figure out how to use the GitHub suggestion API with a multi-line diff):

```
      final List<String> allOutputLines = allOutput.split('\n');
      if (allOutputLines.length > 10) {
        allOutput = <String>[
          ...allOutputLines.take(10),
          '(more stdout/stderr lines truncated)',
        ].join('\n');
      }
```

Reply:
This is a nit suggestion, approving regardless whether you take it

Reply:
I figured if there's that much output, I have no way to guess if the useful part is at the top or bottom (or neither) so...

Reply:
You're right, I figured either way you have a 50% chance you got something interesting :)